ci-operator/jobs/openshift/kubernetes-autoscaler/openshift-kubernetes-autoscaler-presubmits.yaml

```diff
@@ -32,35 +32,12 @@ presubmits:
             cpu: 10m
       serviceAccountName: ci-operator
     trigger: ((?m)^/test( all| unit),?(\s+|$))
-  - agent: kubernetes
+  openshift/online-registration:
+  - agent: jenkins
     always_run: true
-    branches:
-    - release-3.10
-    context: ci/prow/unit
-    decorate: true
-    name: pull-ci-kubernetes-autoscaler-unit-3.10
-    rerun_command: /test unit
-    skip_cloning: true
-    spec:
-      containers:
-      - args:
-        - --artifact-dir=$(ARTIFACTS)
-        - --target=unit
-        - --target=[images]
-        command:
-        - ci-operator
-        env:
-        - name: CONFIG_SPEC
-          valueFrom:
-            configMapKeyRef:
-              key: release-3.10.yaml
-              name: ci-operator-kubernetes-autoscaler
-        image: ci-operator:latest
-        name: test
-        resources:
-          limits:
-            cpu: 500m
-          requests:
-            cpu: 10m
-      serviceAccountName: ci-operator
-    trigger: ((?m)^/test( all| unit),?(\s+|$))
+    context: ci/openshift-jenkins
+    labels:
+      master: ci.dev.openshift.redhat.com
+    name: test_branch_autoscaler
```

Comment:
This gets run over `openshift/online-registration` repo, not `openshift/kubernetes-autoscaler`